app/views/runs/show.html.erb

```diff
@@ -43,11 +43,27 @@
         <tr>
           <td class="testaction-toggle run-test-name-column cursor-hand"><%= run_test.name %></td>
           <td class="testaction-toggle run-test-description-column"><%= run_test.description %></td>
-          <td class="run-test-success-column"><%= display_test_status(run_test) %><div class="clearfix"></div></td>
+          <td class="run-test-success-column"><%= display_test_status(run_test) %><%= carousel_playback_button(run_test, run_test.id) %><div class="clearfix"></div></td>
         </tr>
         <tr class="testaction-row">
-          <td colspan='3'>
+        <%- run_browser_ids(run_test.id).map do |browser| %>
+          <%= tag("div", class: ["outer-scroll-#{browser}"], id: ["outer-scroll-#{browser}"], style: ["display: none"]) %>
+          <%= tag("div", class: ["scroller"], style: ["margin: 0 auto;"]) %>
+                <%- for @screenshot in run_test.screenshots(browser) %>
+                  <%- for @base64string in @screenshot %>
+                    <%= tag("div", style: ["margin:0 auto;"]) %>
+                    <img src="data:image/png;base64,<%= @base64string %>" align= "center">
+                   </div>
+                  <%- end %>
+                <%- end %>
+            </div>
+            </div>
```

Comment:
I think we can remove a nesting level here. And can we move `margin: 0; auto` to a class like "margin-zero-auto"? And IDs should be unique.
```erb
<% i = 1 %>
<%- run_browser_ids(run_test.id).map do |browser| %>
  <%= tag("div", class: ["outer-scroll-#{browser}"], id: ["outer-scroll-#{browser}-#{i}"], style: ["display: none"]) %>
    <%= tag("div", class: ["scroller", "margin-zero-auto"]) %>
    <%- run_test.screenshots(browser).each dp |screenshot| %>
      <%= tag("div", class: ["margin-zero-auto"]) %>
        <img src="data:image/png;base64,<%= screenshot.base64string %>" align= "center">
      </div>
    <%- end %>
    </div>
  </div>
  <% i += 1 %>
<%- end %>
```